akka-docs/src/main/paradox/persistence-schema-evolution.md

```diff
@@ -1,5 +1,17 @@
 # Persistence - Schema Evolution
 
+## Dependency
+
+To use Akka Persistence - Schema Evolution, you must add the following dependency in your project.
```

Comment:
again, this is not something one uses; it is only a docs page. This should explain "This documentation page touches upon Akka Persitence... and some concepts are applicable to any persistence or messaging as well